bsp/smartfusion2/drivers/drv_gpio.c

```diff
@@ -0,0 +1,434 @@
+#include <rtthread.h>
```

Comment:
Please provide the copyright information.

Reply:
I've finished the revision.
- [add support for Microsemi SmartFusion2 family FPGA](https://github.com/RT-Thread/rt-thread/pull/3661/commits)